### DISCUSSION TITLE
[Cloud Engineer] Optimize software and architecture for asynchronous and scheduled jobs

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#34] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/optimize-software-and-architecture-for-asynchronous-and-scheduled-jobs.html

Intent: Use efficient software designs and architectures to minimize the average resources required per unit of work. Implement mechanisms that result in even utilization of components to reduce resources that are idle between tasks and minimize the impact of load spikes.

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/64)
